pkg/bgpv1/gobgp/server.go

```diff
@@ -222,9 +222,10 @@ func (g *GoBGPServer) getPeerConfig(ctx context.Context, n *v2alpha1api.CiliumBG
 	peer.Timers.Config = &gobgp.TimersConfig{
 		// If any of the timers is not set (zero), it will be defaulted at the gobgp level.
 		// However, they should be already defaulted at this point.
-		ConnectRetry:      uint64(n.ConnectRetryTime.Round(time.Second).Seconds()),
-		HoldTime:          uint64(n.HoldTime.Round(time.Second).Seconds()),
-		KeepaliveInterval: uint64(n.KeepAliveTime.Round(time.Second).Seconds()),
+		ConnectRetry:           uint64(n.ConnectRetryTime.Round(time.Second).Seconds()),
+		HoldTime:               uint64(n.HoldTime.Round(time.Second).Seconds()),
+		KeepaliveInterval:      uint64(n.KeepAliveTime.Round(time.Second).Seconds()),
+		IdleHoldTimeAfterReset: uint64(types.DefaultIdleHoldTimeAfterReset.Seconds()), // const in bgpv1/types/bgp.go
```

Comment:
As this timer config is GoBGP-specific (it applies it only when `ResetPeer` GRPC is called), I would say we should keep it in the `gobgp` package.